docs/walkthrough/login.md

```diff
@@ -2,17 +2,17 @@
 # Login
 
 Since Finetuner leverages cloud resources for fine-tuning,
-you are required to {meth}`~finetuner.login()` (or {func}`~finetuner.notebook_login()`) and obtain a token from Jina before starting a fine-tuning job.
+you are required to {meth}`~finetuner.login()` and obtain a token from Jina before starting a fine-tuning job.
 It is as simple as:
 
 ```python
 import finetuner
 
-finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
+finetuner.login()
 ```
 
 A browser window should pop up with different login options.
-After {meth}`~finetuner.login()` or {func}`~finetuner.notebook_login()` you will see this in your terminal:
```

Comment:
It would be nice to have documentation for `login(interactive=True)`, but we can do it later

Reply:
it is actually not recommended, we implemented in the way that we can automatically detect a notebook @guenthermi

Reply:
ah ok good!